app/durableObjects/ChatRoom.server.ts

```diff
@@ -34,9 +34,9 @@ export class ChatRoom extends Server<Env> {
 			)
 		)
 
-		if (!this.ctx.storage.getAlarm()) {
+		if (!(await this.ctx.storage.getAlarm())) {
```

Comment:
fml. this would have been caught with a lint rule. I'll work on adding a better lint setup soon.